gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/ProfileCard.kt

```diff
@@ -25,9 +25,19 @@ import com.gravatar.ui.GravatarTheme
 import com.gravatar.ui.components.ComponentState
 import com.gravatar.ui.components.ProfileSummary
 import com.gravatar.ui.components.atomic.Avatar
+import com.gravatar.ui.components.atomic.ViewProfileButton
 
 @Composable
-internal fun ProfileCard(profile: ComponentState<Profile>?, modifier: Modifier = Modifier) {
+internal fun ProfileCard(
+    profile: ComponentState<Profile>?,
+    modifier: Modifier = Modifier,
+    viewProfile: @Composable ((state: ComponentState<Profile>) -> Unit) = { profileState ->
+        ViewProfileButton(
+            state = profileState,
+            modifier = Modifier.height(32.dp),
+        )
+    },
```

Comment:
ohh it's a leftover. I initially removed the `ViewProfile` for Empty profiles as per this design, but since we have this button for non-empty profiles I kept it for empty as well.

<img width="250" alt="Screenshot 2024-12-18 at 10 12 36" src="https://github.com/user-attachments/assets/b02d9212-2596-4bb2-9d25-7e50655e3d96" />

Reply:
Should be good now.